app/services/create_household.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# Ultimately, creates a household for a user
+# A member object is created for the user object in order
+# to associate a user to a household. 
+class CreateHousehold
+    attr_reader :household
+  
+    def initialize(name:, number:, address:, current_user:)
+        @member = current_user.create_member(number: 1,
+                                            first_name: current_user.first_name,
+                                            middle_name: current_user.middle_name,
+                                            last_name: current_user.last_name,
+                                            date_of_birth: current_user.date_of_birth,
+                                            email: current_user.email,
```

Comment:
I'm not sure of the rules behind this, but does this mean email is required? I think we'll have a high completion rate, but someone doesn't necessarily need to submit an email to have a record created in FreshTrak.

Reply:
Hi David, this is good to know! This is some of the logic that will be dialed back for the pull request since this PR should primarily focus on households not the member entity.